### PR TITLE
Make Windows default configuration build shared library 

### PR DESCRIPTION
**Pull request by rabliyev**
This is required in order to use metee with C# / Python

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Cross-platform access library for Intel(R) CSME HECI interface.
 ## CMake Build
 
 ME TEE library uses CMake for both Linux and Windows builds.
+Adapted to work with managed code: C# and Python
```
Comment (tomasbw) [inline README.md:8]: What does it mean?

File: win32.cmake [hunk 2]
```diff
@@ -6,7 +6,7 @@ set(TEE_SOURCES
     src/Windows/metee_winhelpers.c
 )
 
-add_library(${PROJECT_NAME} ${TEE_SOURCES})
+add_library(${PROJECT_NAME} SHARED ${TEE_SOURCES})
```
Comment (tomasbw) [inline win32.cmake:9]: This is used statically in LMS and other projects, maybe you want to create both libraries....

**Comment by rabliyev** [comment]
This is mistake in the pull request.
It was intended to my fork not this repo

File: win32.cmake (hunk 2 above)
Comment (ausyskin) [reply win32.cmake:9]: No need for that, there is BUILD_SHARED_LIBS configuration flags

**Comment by rabliyev** [comment]
Nice
Missed this in the documentation

**Pull request closed without merging**